packages/shared/src/modules/local-storage/LocalStorage.ts

```
@@ -0,0 +1,11 @@
+import { LocalStorageKeyType } from '.';
+
+export default class LocalStorage {
+  static get(key: LocalStorageKeyType) {
+    return JSON.parse(localStorage.getItem(key) ?? '');
+  }
+
+  static set(key: LocalStorageKeyType, value: unknown) {
+    localStorage.setItem(key, JSON.stringify(value));
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)